# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/svg/filters/css-filter-chains/reftest.list
# Commit: 8c70bc92ad36
# Full Hash: 8c70bc92ad36beb61f2cd6031e80ace3793fe53b
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 1 - Improve how WR handles bounding rects for off-screen surfaces r=gfx-reviewers
#   
#   This patch introduces a number of subtle but important changes
#   to how we deal with off-screen surfaces. The overall goals are:
#   
# ==============================================================================

diff -r c6f1243c3ba4 -r 8c70bc92ad36 layout/reftests/svg/filters/css-filter-chains/reftest.list
--- a/layout/reftests/svg/filters/css-filter-chains/reftest.list	Wed Feb 23 20:42:19 2022 +0000
+++ b/layout/reftests/svg/filters/css-filter-chains/reftest.list	Wed Feb 23 20:49:26 2022 +0000
@@ -2,6 +2,6 @@
 # e.g. filter: blur(3px) grayscale(0.5) invert(0.2);
 
 # Some platforms render this complex filter chain a little differently, and that's ok.
-fuzzy(4-6,12000-19484) fuzzy-if(swgl,5-10,13600-20088) == long-chain.html long-chain-ref.html # Win10: Bug 1258241
+fuzzy(4-6,12000-19950) fuzzy-if(swgl,5-10,13600-20260) == long-chain.html long-chain-ref.html # Win10: Bug 1258241
 == moz-element.html moz-element-ref.html
 fuzzy-if(!useDrawSnapshot,13-15,7670-7982) fuzzy-if(!useDrawSnapshot&&swgl,11-12,14052-14056) == same-filter.html same-filter-ref.html